config/initializers/oauth2.rb

```diff
@@ -1,17 +0,0 @@
-OAuth2::MACToken.prepend(Module.new do
```

Comment:
This was removed in `oauth2` v2, this most likely was not even used.

Related: https://github.com/3scale/porta/pull/2971/files#r892407407, https://github.com/oauth-xx/oauth2/pull/587